filiptronicek/node-ovsx-sign
# zip it up

Due to recent changes in VS Code, we now need to make sure that `extension.sigzip` is an unzippable zip file containing at least a `.signature.p7s` file.

Because of this, `sign` now produces a zip archive (`extension.sigzip`), inside which there is an empty `.signature.p7s` and `.signature` containing the actual signature.

## How to test

```
yarn build
node lib/node-ovsx-sign sign extension.vsix private.pem
node lib/node-ovsx-sign verify -p public.pem extension.vsix extension.sigzip
```

The above should produce an `extension.sigzip` and after unzipping should contain two files: `.signature.p7s` and `.signature`. The last command should return `Signature is valid`.
